modules/apps/27-interchain-accounts/keeper/keeper.go

```diff
@@ -57,28 +57,18 @@ func NewKeeper(
 	}
 }
 
-// SerializeCosmosTx marshals data to bytes using the provided codec
-func (k Keeper) SerializeCosmosTx(cdc codec.BinaryCodec, msgs []sdk.Msg) ([]byte, error) {
-	msgAnys := make([]*codectypes.Any, len(msgs))
+// SerializeCosmosTx packs a slice of sdk.Msg into a slice of Any's.
+func (k Keeper) SerializeCosmosTx(cdc codec.BinaryCodec, msgs []sdk.Msg) (msgAnys []*codectypes.Any, err error) {
```

Comment:
I need to update this to return the CosmosTx bytes. Should be called by ica-auth modules